psydac/fem/splines.py

```diff
@@ -82,7 +82,11 @@ def __init__( self, degree, knots=None, grid=None, multiplicity=None, parent_mul
 
         if multiplicity is None:multiplicity = 1
                 
-        if parent_multiplicity is None:parent_multiplicity = 1
+        if parent_multiplicity is None:
+            if multiplicity is not None:
+                parent_multiplicity = multiplicity
+            else:
+                parent_multiplicity = 1
 
         assert parent_multiplicity >= multiplicity
 
```

Comment:
Should we not raise an error if both `knots` and `multiplicity` are given?

Reply:
You are right, added in [68937d4](https://github.com/pyccel/psydac/pull/244/commits/68937d4ffbcceec954f6f2e2bef2287f88f62853)